api/README.md

```diff
@@ -123,41 +180,54 @@ pushd ../engines/router
 popd
 ```
 
+#### <b>Start Turing API</b>
+---
 Now fire up the turing API server (as a daemon or tmux or some other terminal process).
 
 ```
-go run turing/cmd/main.go -config=config-dev.yaml
+go run turing/cmd/main.go -config=config-dev.yaml -config=config-dev-exp-engine.yaml
```

Comment:
In general, it may be nice to double-check all the paths in this README file, to see if they make sense regardless of if the previous steps in the README are run or no (so `pushd/popd` or `cd ${path} && ...` may be a good idea to capture the steps rather than standalone `cd` commands).

Reply:
Do you think we can have a single make target at the repo root that's just `e2e-local` or something which will do all of these actions described in this README at one shot? The individual commands and explanation in this README are useful but it may be a lot easier to get started if we can wrap everything into a single command to be run by the user. wdyt?

Reply:
At best, it'll do these as we'll need to run the API server separately after these steps prior to triggering the test, do you think its useful to have this?
```
.PHONY: all-local-test-steps
all-local-test-steps:
	cd ../engines/experiment/examples/plugins/hardcoded && make proprietary-exp-plugin GOOS=darwin GOARCH=arm64
	cd ../engines/experiment/examples/plugins/hardcoded && make build-local-proprietary-exp-plugin-image
	cd ../engines/router && make build-local-router-image
```

Reply:
Can we also add the following to the target:
* Setting up local infra
* Starting of the API server (this can be a background process)
* Running the tests

It would be nice to include a clean-up-local-e2e target as well, to remove any running processes (such as the API server) and to do docker-compose down on the infra.

Reply:
Simplified the local E2E workflow.